### PR TITLE
Update playbook with new release process

### PR DESCRIPTION
**Pull request by keertk**
@meteorcloudy  here's a first pass at updating the playbook with the new process we talked about. Could you please take a look?
Thanks!
cc @hvadehra

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -51,26 +49,32 @@ an example.
...
+            "https://mirror.bazel.build/bazel_java_tools/release_candidates/java/v11.9/java_tools-v11.9-rc1.zip",
+         ],
+     )     
      ```
      
-  * Repeat for `remote_java_tools_linux`, `remote_java_tools_windows`, `remote_java_tools_darwin_x86_64` and `remote_java_tools_darwin_arm64`
-  * See [#16865](https://github.com/bazelbuild/bazel/pull/17802) for reference ([this](https://github.com/bazelbuild/bazel/pull/17802/commits/8678754e0b1c19269f4974d80ef66e8aeb7d4fb3) commit)
+  * Repeat for `remote_java_tools_linux`, `remote_java_tools_windows`, `remote_java_tools_darwin_x86_64` and `remote_java_tools_darwin_arm64`    
+
+6. Edit [distdir_deps.bzl](https://github.com/bazelbuild/bazel/blob/master/distdir_deps.bzl#L65) in the Bazel repository by updating the `archive`, `sha256`, and `urls` fields with the updates made to the new branch (step 5). Create a new pull request - **example PR coming soon**. The PR triggers the CI presubmit.
```
Comment (meteorcloudy) [inline docs/release.md:71]: Basically, you can point to the latest commit of that branch, you'll need to specify the `strip_prefix` attribute, you can use the following as an example:
```
"archive": "56220853b2d75c478dd1ee71ba7cd0e46363700f.tar.gz",
"sha256": "0f65c471b99c79e97dd18a3571d3707b4dbfc31ff8e9bf7083a09aae0adb7b5e",
"strip_prefix": "rules_java-56220853b2d75c478dd1ee71ba7cd0e46363700f",
"urls": ["https://github.com/meteorcloudy/rules_java/archive/56220853b2d75c478dd1ee71ba7cd0e46363700f.tar.gz"],
```

**Review by meteorcloudy — APPROVED** [review]
Thanks!

File: docs/release.md (hunk 1 above)
Comment (keertk) [reply docs/release.md]: Updated, thanks!

File: docs/release.md [hunk 2]
```diff
@@ -51,26 +49,45 @@ an example.
...
+  * Download the tar.gz file at `https://github.com/bazelbuild/rules_java/archive/<commit hash>.tar.gz`
+  * Run `shasum -a 256 <file>`
+  * Update the following fields (note: add `strip_prefix`)
+    
+     Example:
+     ```starlark
+     "archive": "56220853b2d75c478dd1ee71ba7cd0e46363700f.tar.gz",
+     "sha256": "0f65c471b99c79e97dd18a3571d3707b4dbfc31ff8e9bf7083a09aae0adb7b5e",
+     "strip_prefix": "rules_java-56220853b2d75c478dd1ee71ba7cd0e46363700f",
+     "urls": ["https://github.com/meteorcloudy/rules_java/archive/56220853b2d75c478dd1ee71ba7cd0e46363700f.tar.gz"],
```
Comment (meteorcloudy) [inline docs/release.md:83]:
```suggestion
     "urls": ["https://github.com/bazelbuild/rules_java/archive/56220853b2d75c478dd1ee71ba7cd0e46363700f.tar.gz"],
```

**Review by meteorcloudy — APPROVED** [review]

**Review by meteorcloudy — APPROVED** [review]

**Comment by meteorcloudy** [comment]
@hvadehra Can you take a final look?

**Review by hvadehra — APPROVED** [review]
LGTM

**Pull request merged**